tensorflow/java/src/gen/cc/ops/op_generator.cc

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
My mental model (and please correct me if I'm wrong) is that no code in `src/gen` will be part of the `.jar` distributions. The annotation processor will have to be part of it, so it makes sense for that to be in something like `src/main/java/org/tensorflow/annotations/Processor.java` or something.

If that is correct, then I don't think we need the subdirectories here now. I don't yet see the need for any other generators, but if and when they do come about, we can partition the files differently then. For now, I'd be tempted to stick with a simpler/flatter directory structure